generators/generator-bot-core-language/generators/app/templates/README.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to your new bot
 
-This Bot Project was created using the Conversational Core template, and contains support for a base set of conversational flows.
+This Bot Project was created using the Core bot with Azure Language Understanding template, and contains support for a base set of conversational flows.
```

Comment:
nit: Core Bot (upper-case B)

Reply:
this is the casing design wants.

Reply:
In that case we should fix the casing elsewhere to be consistent.

Reply:
But suggesting that it would be lowercase here goes against the guidance shared on my comment earlier today:
> When it's referring to a specific product, e.g. Empty Bot or Core Bot with Azure... we use capital casing. for sentences that refer to a generic bot, we use lowercase.

What we have now is inconsistent from doc to doc, we should lean on the [Microsoft Writing Style Guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33685) which says:
> Microsoft style uses sentence-style capitalization. That means everything is lowercase except the first word and proper nouns, which include the names of brands, products, and services.

Using that definition this sentence would read "_This bot project was created using the Core Bot with Azure Language Understanding template..._"

Reply:
I agree.  I've updated all the generated READMEs to use "bot project".  and I've also updated them to use "Noun Bot ..."